fortify-issue-manager/src/main/java/com/microfocus/security/automation/fortify/issue/manager/FortifyIssueManager.java

```diff
@@ -228,7 +228,7 @@ private ScriptEngine getBugPayloadScript(final String scriptFile)
             if (StringUtils.isEmpty(getPayloadScript)) {
                 throw new ScriptNotFoundException("Script getPayload not found.");
             }
-            final ScriptEngine engine = new ScriptEngineManager().getEngineByName("nashorn");
+            final ScriptEngine engine = new ScriptEngineManager().getEngineFactories().get(0).getScriptEngine();
```

Comment:
I don't strongly object to this but wouldn't using `.getEngineByName("graal.js")` be clearer?